docs: screenshot and copy updates - projects (#4240)
mesosphere/dcos-docs-site: pages/dkp/kommander/2.2/projects/project-quotas-limit-range/index.md

Project Quotas and Limit Ranges can be set up to limit the number of resources the Project team uses. Quotas and Limit Ranges are applied to all project clusters.

> 1. Select the workspace your project was created in from the workspace selection dropdown in the header.

Reviewer: I am not sure if this is the best way to word it, but I think it might a bit clearer
```suggestion
1. Select the workspace that corresponds to the project you created from the selection dropdown in the header.
```

> 1. In the sidebar menu select **Projects**, and then select the project you would like to configure from the table.

Reviewer: ```suggestion
1. In the sidebar menu, select **Projects**. 

1. Select the project you would like to configure from the table.
```

1. Select the **Quotas & Limit Ranges** tab, and then select the **Edit** button.

>     Kommander provides a set of default resources for which you can set Quotas, or you can define Quotas for custom resources. We recommended that you set Quotas for CPU and Memory. Using Limit Ranges, you can restrict the resource consumption of individual Pods, Containers, and Persistent Volume Claims in the project namespace. You can also constrain memory and CPU resources consumed by Pods and Containers, and you can constrain storage resources consumed by Persistent Volume Claims.

Reviewer: ```suggestion
    Kommander provides a set of default resources for which you can set Quotas. Alternatively, you can define Quotas for custom resources. We recommend that you set Quotas for CPU and Memory. By using Limit Ranges, you can restrict the resource consumption of individual Pods, Containers, and Persistent Volume Claims in the project namespace. You can also constrain memory and CPU resources consumed by Pods and Containers, and storage resources consumed by Persistent Volume Claims.
```

Author: So I think you can actually set quotas for default resources, as well as custom resources. It doesn't have to be one or the other. What do you think of this for the first sentence instead:

`Kommander provides a set of default resources for which you can set Quotas. You can also define Quotas for custom resources.`

1. To add a custom quota, scroll to the bottom of the form and select **Add Quota**.

1. When you are finished, select the **Save** button.

> ## Creating Project Quotas & Limit Ranges - CLI Method

Reviewer: ```suggestion
## Create Project Quotas & Limit Ranges - CLI Method
```

Project Quotas and Limit Ranges can be set up to limit the number of resources the Project team uses. Quotas and Limit Ranges are applied to all project clusters.

1. Select the workspace your project was created in from the workspace selection dropdown in the header.

1. In the sidebar menu, select **Projects**. 

> 1. Select the project you would like to configure from the table.

Reviewer 2: ```suggestion
1. Select the project you want to configure from the table.
```
